src/Root.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC } from 'react';
+import { HashRouter, Navigate, Route, Routes } from 'react-router-dom';
+
+import { App } from './App';
+
+import { HomePage } from './pages/HomePage';
+import { PeoplePage } from './pages/PeoplePage';
+import { NotFoundPage } from './pages/NotFoundPage';
+
+export const Root: FC = () => {
+  return (
+    <HashRouter>
+      <Routes>
+        <Route path="/" element={<App />}>
+          <Route index element={<HomePage />} />
+          <Route path="home" element={<Navigate to="/" replace />} />
+
+          <Route path="people">
+            <Route index element={<PeoplePage />} />
+            <Route path=":personSlug" element={<PeoplePage />} />
```

Comment:
Here you are using the same component for two different routes. While this is not necessarily wrong, it might cause confusion and unexpected behavior. If 'PeoplePage' is supposed to display different content based on whether 'personSlug' is present, consider separating these into different components or handling the route parameters in a more clear way inside the 'PeoplePage' component.